### DISCUSSION TITLE
New realistic models for Fooocus

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2355] [Q&A] [answered]
Help me figure out where I can find (чтобы скачать) realistic models for fooocus, what format is needed, and what format to put in which folder. Interested in models for creating cars and buildings (backgrounds for cars).
Fooocus doesn’t want to do what is needed, Dall-e, for example, understands the first time but creates less realistic photos.
![image - 2024-02-25T224440 466](https://github.com/lllyasviel/Fooocus/assets/111929719/c3296459-4c37-4cc0-b9f1-a978dff315e7)

Thank you

**Comment by mashb1t** [comment, marked as answer]
Feel free to use `--preset realistic` or any SDXL model file from e.g. https://civitai.com/search/models?baseModel=SDXL%201.0&sortBy=models_v5&query=realistic. Downloaded files can be put in the models/checkpoints folder and after reloading the models they can be selected in Fooocus.
